addons/sale_loyalty/tests/test_loyalty_history.py

```diff
@@ -0,0 +1,90 @@
+from odoo.fields import Command
+from odoo.tests.common import tagged
+
+from odoo.addons.sale_loyalty.tests.common import TestSaleCouponCommon
+
+
+@tagged('post_install', '-at_install')
+class TestLoyatyhistory(TestSaleCouponCommon):
+
+    def setUp(self):
+        super().setUp()
```

Comment:
Please override setUpClass instead of setUp to avoid triggering the setup once by test method.